From ba595c7337f9d2f5dd6da67c309f3eceba7e76a7 Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Fri, 28 Aug 2026 14:11:05 -0400
Subject: [PATCH] Do not install vir-simd as part of gnuradio4-core.

This is installed from the vir-simd package. Installing it from two
different places is not a good thing to do. This is the way to non
determinstic builds.

Upstream-Status: Pending

Signed-off-by: Philip Balister <philip@balister.org>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 33a95e6..c58b63b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -570,7 +570,6 @@ add_library(gnuradio4::vir ALIAS vir)
 target_include_directories(vir INTERFACE $<BUILD_INTERFACE:${vir-simd_SOURCE_DIR}/>
                                          $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
 install(TARGETS vir EXPORT gnuradio4Targets)
-install(DIRECTORY ${vir-simd_SOURCE_DIR}/vir DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 # the script configure won't run on win32 without asking bash to do it
 if(WIN32)
-- 
2.55.0

